### PR TITLE
feat: implemented search

### PR DESCRIPTION
**Pull request by iswarya15**

### CODE REVIEW

File: team-app/src/Body.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+import Card from "./Card";
+import { teammates } from "./data";
+import Search from "./Search";
+
+const Body = () => {
+  console.log(teammates);
```
Comment (nitishnivedan) [inline team-app/src/Body.js]: remove console

File: team-app/src/Search.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      filteredMembers.length > 0
+        ? setFilteredCards(filteredMembers)
+        : setErrorMessage("Member not found");
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  const filterMembers = () => {
+    return teammates.filter((member) => {
```
Comment (nitishnivedan) [inline team-app/src/Search.js]: can be returned directly without return and curly braces

File: team-app/src/Search.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+
+export default Search = ({ setFilteredCards, teammates }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const handleSubmit = (event) => {
+    try {
```
Comment (nitishnivedan) [inline team-app/src/Search.js:8]: i m just curious to see try, catch here. I need to research. is there a possibility for an exception here?

File: team-app/src/Search.js (hunk 2 above)
Comment (iswarya15) [reply team-app/src/Search.js]: my bad. will remove that.

File: team-app/src/Search.js (hunk 3 above)
Comment (iswarya15) [reply team-app/src/Search.js:8]: added to check if invalid input breaks the code while filtering.

File: team-app/src/Body.js (hunk 1 above)
Comment (iswarya15) [reply team-app/src/Body.js]: removed

File: team-app/src/card.css [hunk 4]
```diff
@@ -20,8 +20,9 @@
 }
 
 .id {
-    background-color: yellow;
+    background-color: rgb(255, 255, 93);
```
Comment (ap221882) [inline team-app/src/card.css:23]: Don't name CSS class selectors as id, class, h1, body, etc. Name them according to their use cases.

File: team-app/src/Search.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      event.preventDefault();
+      setErrorMessage("");
+
+      if (searchInput != "" && searchInput.length < 3) {
+        setErrorMessage("Please enter at least 3 letters");
+        return;
+      }
+
+      const filteredMembers = teammates.filter(
+        (member) =>
```
Comment (ap221882) [inline team-app/src/Search.js:18]: Try to keep this reusable function out of the component and import it in the component then

**Review by ap221882 — COMMENTED** [review]
Nicely written. Add a screenshot or a video of the working project with your PR next time. It is nice to add.

**Review by shaksham08 — APPROVED** [review]
Agree to the comments added by Ajay , apart from it all looks good to me

File: team-app/src/card.css (hunk 4 above)
Comment (iswarya15) [reply team-app/src/card.css:23]: Updated.

**Pull request merged**